tests/foreman/cli/test_import.py

```diff
@@ -19,33 +19,34 @@
 from robottelo.cli.user import User
 
 
-def csv_to_dataset(csv_file):
+def csv_to_dataset(csv_files):
     """Process a remote CSV file.
 
-    Read a remote CSV file, process it and return it.
+    Read the remote CSV files, and return a list of dictionaries for them
 
-    :param csv_file: A string. The path to a CSV file that resides
+    :param csv_files: A list of strings. The paths to CSV files that reside
         on a remote server.
 
-    :returns: A dictionary holding the contents of the CSV file.
+    :returns: A dictionary holding the contents of the CSV files.
 
     """
-    ssh_cat = ssh.command('cat {0}'.format(csv_file))
-    if ssh_cat.return_code != 0:
-        raise Exception(ssh_cat.stderr())
-    else:
-        ssh_cat.stdout.pop()
-        csv = ssh_cat.stdout
-    keys = csv[0].split(',')
-    del csv[0]
-    return [
-        dict(zip(keys, val.split(',')))
-        for val
-        in csv
-    ]
-
-
-def build_csv_file(rows=None):
+    result = []
+    for csv_file in csv_files:
+        ssh_cat = ssh.command(u'cat {0}'.format(csv_file))
+        if ssh_cat.return_code != 0:
+            raise Exception(ssh_cat.stderr())
```

Comment:
Also this seems to be an helper method, maybe it should be moved to another module.